src/components/list/list-renderer.tsx

```diff
@@ -70,7 +69,7 @@ export class ListRenderer {
         return (
             <ul
                 class={classNames}
-                aria-hidden={(!this.config.isOpen).toString()}
+                aria-hidden={(this.config.type === 'menu').toString()}
```

Comment:
This doesn't seem quite right. Won't the menu list always be hidden from ARIA users?